core.js

```diff
@@ -150,6 +172,10 @@ Velocity = Velocity || {};
      * @param options.sampleTestGenerator.contents {String} contents of the test file the path that's returned
      */
     registerTestingFramework: function (name, options) {
+      if (options.regex[0] == '^') {
```

Comment:
I don't like this approach.
Currently the test framework has the full control over the loading path.

I think we should make the regEx relative to the tests directory (either the one in the app root directory or in a package). (This is a breaking change)